Add option for weak verification

knuton opened the pull request:
This adds the option to pass only an assembly path to `signcode.verify` and skip the leaf hash check. This is useful if the only goal is to check for the presence of a signature.

Does that make sense? I was wondering about the status of `verify`; it is an export but not documented or available through the CLI. I could update this PR with documentation and a CLI option if desired.

knuton committed:
Add option for weak verification

Add the option to pass only an assembly path to `signcode.verify` and
skip the leaf hash check. This is useful if the only goal is to check
for the presence of a signature.

kevinsawicki commented:
> I could update this PR with documentation and a CLI option if desired.

This sounds good to me, and thanks for adding this option 👍

knuton committed:
Add verify to CLI and Readme

This changes the `signcode` script to expect one of two commands, 'sign'
or 'verify'. Uses an up-to-date version of yargs to simplify argument
processing in `cli.js`.

knuton commented:
I just pushed a changeset which adds `verify` to both CLI and the docs.

As implemented, this is a breaking change, because signing via the CLI now requires use of the `sign` command. It could be made backwards compatible by assuming `sign` to be the default.

kevinsawicki commented:
> As implemented, this is a breaking change, because signing via the CLI now requires use of the sign command.

That is fine, this is still pre 1.0

kevinsawicki commented:
Thanks for this @knuton 👍

kevinsawicki commented:
Published as 1.0.0

knuton commented:
> Published as 1.0.0

Great, thanks! 😃
